File: backend/Multibanca.Domain.Models/Multibanca/generar_carta_resguardo.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class generar_carta_resguardo : base_auditoria
    {
        public int id_generar_carta_resguardo { get; set; }
        public long id_expediente { get; set; }
        public bool? generar_carta { get; set; }
        public string? tipo_carta { get; set; }
        public bool? enviar_a_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/generar_carta_resguardo/models/generar_carta_resguardo.ts
export interface GenerarCartaResguardo {
  id_generar_carta_resguardo: number;
  id_expediente: number;
  generar_carta: boolean;
  tipo_carta?: string | null;
  enviar_a_reparo: boolean;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
